README.md

```diff
@@ -151,54 +151,67 @@ once every minute or less).
 
 ## Contribute
 
-If you wish to contribute more functions to this package, you are more than
-welcome to do so. Please, fork this project, make your changes following the
-usual [PEP 8][pep8] code style, sprinkling with tests and submit a PR for
-review.
+From a code perspective, if you wish to contribute, you will need to run a
+Python 3.6+ environment. Please, fork this project, write unit tests to cover
+the proposed changes, implement the changes, ensure they meet the formatting
+standards set out by `black`, `flake8`, `isort`, and `mypy`, add an entry into
+`CHANGELOG.md`, and then raise a PR to the repository for review
 
-[pep8]: https://pycodestyle.readthedocs.io/en/latest/
+Please refer to the [formatting](#formatting-and-linting) section for more
+information on the formatting standards.
+
+The Chaos Toolkit projects require all contributors must sign a
+[Developer Certificate of Origin][dco] on each commit they would like to merge
+into the master branch of the repository. Please, make sure you can abide by
+the rules of the DCO before submitting a PR.
+
+[dco]: https://github.com/probot/dco#how-it-works
 
 ### Develop
 
 If you wish to develop on this project, make sure to install the development
 dependencies. But first, [create a virtual environment][venv] and then install
 those dependencies.
 
-[venv]: http://chaostoolkit.org/reference/usage/install/#create-a-virtual-environment
+[venv]: http://docs.chaostoolkit.org/reference/usage/install/#create-a-virtual-environment
```

Comment:
Tried your proposed link and it didn't seem to work for me whereas the original one did

```suggestion
[venv]: http://chaostoolkit.org/reference/usage/install/#create-a-virtual-environment
```

Reply:
That's what I get for copy pasting from other docs. CTK ones are wrong then haha